src/FieldDescription/FieldDescriptionFactoryInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\FieldDescription;
+
+use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
+
+interface FieldDescriptionFactoryInterface
+{
+    /**
+     * @phpstan-param class-string $class
+     */
+    public function create(string $class, string $name, array $options = []): FieldDescriptionInterface;
```

Comment:
Can we use a more explicite name than `$class` ?

Reply:
we could use `modelClass` maybe, but I guess this is "sync" with `AdminInterface::getClass`

Reply:
Can we add some type hints for the `$options`?